libcontainer/specconv/spec_linux.go

```diff
@@ -635,11 +643,17 @@ func parseMountOptions(options []string) (int, []int, string) {
 			}
 		} else if f, exists := propagationFlags[o]; exists && f.flag != 0 {
 			pgflag = append(pgflag, f.flag)
+		} else if f, exists := extensionFlags[o]; exists && f.flag != 0 {
+			if f.clear {
+				extFlags &= ^f.flag
+			} else {
+				extFlags |= f.flag
+			}
 		} else {
 			data = append(data, o)
```

Comment:
I found one here http://linux.die.net/man/1/funionfs.
WDYT about calling it runc_copyup?